examples/eks-msk-opensearch/terraform/iam_policies/logging.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I can't comment on it directly but GitHub is showing that there is a 28MB binary file in the `connectors` directory above. What is that? Why do we need it? Was it added by mistake?